### DISCUSSION TITLE
Compiling under Oracle Linux 9.x fails

### DISCUSSION THREAD

**Discussion by leocb** [dotnet/source-build#3464] [General]
### Describe the Bug

If I try to compile the source under Oracle Linux 9.x at some point I get this error:

```bash
Build FAILED 

/home/lbottaro/source/.dotnet/sdk/6.0.115/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets(129,5): error NETSDK1083: The specified RuntimeIdentifier 'ol.9.0-x64' is not recognized. [/home/lbottaro/source/src/runtime/artifacts/source-build/self/src/Build.proj]
```
and lots of [warning]s - don't know if these are relevant
```bash
[Warning] Detected packages in the cache that should be source-built, but contents don't match. See /home/lbottaro/source/artifacts/conflict-report/before-arcade/ for usage details: [/home/lbottaro/source/repos/arcade.proj]
```

I discussed this with @omajid and apparently there's a PR to fix this already: /dotnet/runtime/pull/81115 - But taking a quick glance at the changed files, it seems OL9 ids are still missing.

Here's the complete build log:
[Build_0517165001.zip](https://github.com/dotnet/source-build/files/11508519/Build_0517165001.zip)

**Comment by MichaelSimons** [comment]
Unfortunately because the RIDs are missing, you will need to follow the [Building for New OS (Using a RID unknown to .NET)](https://github.com/dotnet/source-build/blob/main/Documentation/bootstrapping-guidelines.md#building-for-new-os-using-a-rid-unknown-to-net) instructions.  I will try and push https://github.com/dotnet/runtime/pull/81115 along.

**Comment by MichaelSimons** [comment]
@richlander - another scenario that would benefit from the [proposed RID plan](https://github.com/dotnet/runtime/issues/83246).
